sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1453,7 +1453,18 @@ graph LR
     Recording -->|End Recording| Executing
 ....
 
-==== Queue Properties
+==== Implicit Queue Recording
```

Comment:
Nitpick, I'm not sure if "Transitive" or "Implicit" is a better term to describe this behavior